src/components/Button/Button.module.css

```diff
@@ -11,49 +11,28 @@
  * 4) Adds a gap between children to automatically handle spacing between icons and text
  */
 .button {
+  @mixin eds-theme-typography-button-label;
```

Comment:
All the button sizes use the same typography styles now, so I moved it into the main styles.